### PR TITLE
Chore/ch8820/add avalanche config

### PR DESCRIPTION
**Pull request by PiotrTrzpil**

### CODE REVIEW

File: core/chains/configs.go [hunk 1]
```diff
@@ -210,10 +210,27 @@ func setConfigs() {
...
+	avalancheMainnet := FallbackConfig
+	avalancheMainnet.LinkContractAddress = "0x350a791Bfc2C21F9Ed5d10980Dad2e2638ffa7f6" // TBD
+	avalancheMainnet.EthFinalityDepth = 1
+	avalancheMainnet.GasEstimatorMode = "FixedPrice"
+	avalancheMainnet.EthGasPriceDefault = *big.NewInt(225000000000) // 225 Gwei
+	avalancheMainnet.EthMaxGasPriceWei = *big.NewInt(225000000000)
+	avalancheMainnet.EthMinGasPriceWei = *big.NewInt(225000000000)
+	avalancheMainnet.MinIncomingConfirmations = 1
+	avalancheMainnet.MinRequiredOutgoingConfirmations = 1
+	avalancheMainnet.OCRContractConfirmations = 1
```
Comment (aalu1418) [inline core/chains/configs.go:233]: Would it be good to have this match the layout of other networks in the file?
(where the testnet configs are a modified from the mainnet configs)

**Review by aalu1418 — DISMISSED** [review]
Parameters look good to me! Just a small question on matching code structure of other networks

File: core/chains/configs.go (hunk 1 above)
Comment (PiotrTrzpil) [reply core/chains/configs.go]: good point 👍🏼

**Review by aalu1418 — APPROVED** [review]
👍🏼

**Pull request merged**